components/server/src/api/organization-service-api.ts

```diff
@@ -251,6 +251,20 @@ export class OrganizationServiceAPI implements ServiceImpl<typeof OrganizationSe
         const settings = await this.orgService.getSettings(ctxUserId(), req.organizationId);
         const response = new GetOrganizationSettingsResponse();
         response.settings = this.apiConverter.toOrganizationSettings(settings);
+
+        // resolve the avatar URL for the featured member in the welcome message
+        const onboardingWelcomeMessageFeaturedMemberId =
```

Comment:
Why is this not happening in `orgService.getSettings`?

Reply:
@geropl the reason I opted not to do so is that the resolved avatar URL is a response-specific thing that doesn't exist at all on the protocol. If we did it in `getSettings`, we would also resolve the avatar in all other places we call `orgService.getSettings`, where it's unnecessary.